Source/WebKit/UIProcess/WebPageProxyTesting.cpp

```diff
@@ -111,62 +111,62 @@ bool WebPageProxyTesting::isEditingCommandEnabled(const String& commandName)
 
 void WebPageProxyTesting::dumpPrivateClickMeasurement(CompletionHandler<void(const String&)>&& completionHandler)
 {
-    m_page->websiteDataStore().protectedNetworkProcess()->sendWithAsyncReply(Messages::NetworkProcess::DumpPrivateClickMeasurement(m_page->websiteDataStore().sessionID()), WTFMove(completionHandler));
+    protectedPage()->protectedWebsiteDataStore()->protectedNetworkProcess()->sendWithAsyncReply(Messages::NetworkProcess::DumpPrivateClickMeasurement(m_page->websiteDataStore().sessionID()), WTFMove(completionHandler));
```

Comment:
Hm... it's unfortunate that we have to use protectedPage() just because protectedWebsiteDataStore() isn't an inline function but I suppose this is fine.

Reply:
There are only two possible amounts of refcounts: Too few and too many. There is no known algorithm for the exact right number. Given that choice, we prefer too many :P.